darts/tests/models/forecasting/test_ensemble_models.py

```diff
@@ -763,3 +764,67 @@ def get_global_ensemble_model(output_chunk_length=5):
                 ),
             ],
         )
+
+    @pytest.mark.parametrize("model_cls", [NaiveEnsembleModel, RegressionEnsembleModel])
+    def test_save_load_ensemble_models(self, tmpdir_module, model_cls):
+        # check if save and load methods work and
+        # if loaded ensemble model creates same forecasts as original ensemble models
+        cwd = os.getcwd()
+        os.chdir(tmpdir_module)
+        os.mkdir(model_cls.__name__)
+        full_model_path_str = os.path.join(tmpdir_module, model_cls.__name__)
+        os.chdir(full_model_path_str)
+        kwargs = {}
+        expected_suffixes = [".pkl", ".pkl.RNNModel_2.pt", ".pkl.RNNModel_2.pt.ckpt"]
+
+        if issubclass(model_cls, RegressionEnsembleModel):
+            kwargs["regression_train_n_points"] = 5
+
+        if TORCH_AVAILABLE:
+            model = model_cls(
+                [
+                    LinearRegressionModel(lags=[-1]),
+                    NaiveSeasonal(K=1),
+                    RNNModel(10, n_epochs=1),
```

Comment:
We need to add the tfm kwargs, to make it run on M1 chips
```suggestion
                    RNNModel(10, n_epochs=1, **tfm_kwargs),
```